test/engine/controller/logtail_control.go

```
@@ -36,6 +40,31 @@ func (l *LogtailController) Init(parent *CancelChain, fullCfg *config.Case) erro
 	// todo currently only support one fixed config with cgo
 	if len(fullCfg.Ilogtail.Config) == 1 {
 		cfg := fullCfg.Ilogtail.Config[0]
+		if len(cfg.Detail) != 0 {
+			_ = os.Remove(config.ConfigYamlFileDir)
+			_ = os.Mkdir(config.ConfigYamlFileDir, 0750)
```

don't ignore exception when init